src/main/scala/firrtl/backends/verilog/LegalizeVerilog.scala

```diff
@@ -47,16 +48,44 @@ object LegalizeVerilog extends Pass {
     }
   }
 
-  import firrtl.passes.PadWidths.forceWidth
   private def getWidth(e: Expression): Int = bitWidth(e.tpe).toInt
 
+  /* Verilog has the width of (a % b) = Max(W(a), W(b))
+   * FIRRTL has the width of (a % b) = Min(W(a), W(b)), which makes more sense,
+   * but nevertheless is a problem when emitting verilog
```

Comment:
"which makes more sense" 👀

Reply:
I just copied the comment from here 🙃 https://github.com/chipsalliance/firrtl/blob/c56e341d25e35a5207c4eea12bac9bf0ddd8b652/src/main/scala/firrtl/passes/VerilogModulusCleanup.scala#L16